src/org/zaproxy/zap/extension/ascanrulesBeta/SourceCodeDisclosureSVN.java

```diff
@@ -48,6 +50,10 @@
  */
 public class SourceCodeDisclosureSVN extends AbstractAppPlugin {
 
+	private List<Integer> UNWANTED_CODES= new ArrayList<Integer>(7);
```

Comment:
This could/should be a constant (and should be initialised in a static block). Otherwise it will be created a list per instance, which happens often when scanning.